### PR TITLE
Primary changes

### PR DESCRIPTION
**Pull request by OmidCharrakh**

### CODE REVIEW

**Review by HeidiSeibold — COMMENTED** [review]
What does the file .DS_Store do?

File: README.md [hunk 1]
```diff
@@ -20,3 +20,65 @@ Please check out our [contributing guidelines](CONTRIBUTING.md).
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/teaching-data-science/course-template/binder)
 
 
+
+## Gatsby-Ubuntu Instructions:
```
Comment (HeidiSeibold) [inline README.md]: Thanks for that. :clap: 

Can we maybe move this to `CONTRIBUTING.md`?

File: chapters/chapter1.1.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
```
Comment (HeidiSeibold) [inline chapters/chapter1.1.md]: I think having `.` in file names is not such a great idea. Could you rename this maybe?

`git mv chapter1.1.md chapter1-1.md`

**Review by HeidiSeibold — COMMENTED** [review]
As I understand it, slides should be in `intro2ml/static/` (probably best in a subfolder `slides`)

File: chapters/chapter1.1.md (hunk 2 above)
Comment (HeidiSeibold) [reply chapters/chapter1.1.md]: Now that I think about it: Let's use more meaningful names. Use the same naming conventions as in slides (just replace `slides` with `chapter`?):

`git mv chapter1.1.md chapter-basics-whatisml.md`

File: chapters/chapter1.1.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+</exercise>
+
+<exercise id="3" title="Demo">
+
+<iframe width="100%" height="480" src="https://jjallaire.github.io/deep-learning-with-r-notebooks/notebooks/2.1-a-first-look-at-a-neural-network.nb.html"></iframe>
+
+</exercise>
+
+<exercise id="4" title="Quiz">
```
Comment (HeidiSeibold) [inline chapters/chapter1.1.md]: Seems like I get a "that's correct" if I check *any* of the correct answers. What we want would be that I only get it if I check *all* of the correct answers. Can you find out if that is possible?

**Comment by OmidCharrakh** [comment]
> What does the file .DS_Store do?

I think this is not an important change since it is a kind of hidden file that has been created automatically.

File: README.md (hunk 1 above)
Comment (OmidCharrakh) [reply README.md]: Done!

File: chapters/chapter1.1.md (hunk 2 above)
Comment (OmidCharrakh) [reply chapters/chapter1.1.md]: Done!

File: chapters/chapter1.1.md (hunk 3 above)
Comment (OmidCharrakh) [reply chapters/chapter1.1.md]: I searched on the internet and did not find any solution for that. Do you have any idea how I can resolve this issue? In particular, can you please inform me of the GitHub source file for the quiz in the following link?

https://compstat-lmu.shinyapps.io/tutorial-ml-basics/#section-data-and-tasks

Comment (HeidiSeibold) [reply chapters/chapter1.1.md]: Ah yes, I should have provided this in the beginning: https://github.com/compstat-lmu/lecture_i2ml_learnr_tutorials

Not sure if this helps with the issue, but let's see ;)

Comment (OmidCharrakh) [reply chapters/chapter1.1.md]: Dear Heidi, during the last days, I implemented several solutions to fix this issue, but I could not resolve it. Although my methods were unsuccessful, I think you would be better to know them; perhaps another person can utilize each of these candidate options and resolve the issue.

**1. The first option** is to find of way of adding a code chunk (containing R codes) to our .md files. The following link provides an example of this option:

https://rstudio.github.io/learnr/questions.html


**2. The second option** is to like what the following link suggests. To use this option, one has to define <choiceresponse> and <checkboxgroup>.

https://raw.githubusercontent.com/edx/edx-platform/0438878ab96e8be21c4f75fa3979c4eb5b491543/common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

**3. The third possibility** is to re-define the structure of our quizzes in a way that each quiz has only one correct choice. While I am not sure whether we can call this as a "real" option, I noticed that they had used this option in the following course:

https://course.spacy.io/en/

https://raw.githubusercontent.com/ines/spacy-course/master/chapters/en/chapter3.md

**Comment by HeidiSeibold** [comment]
When do you store pdfs in folder `public`, when in `static`?

**Comment by OmidCharrakh** [comment]
> When do you store pdfs in folder `public`, when in `static`?

As far as I understand, we do not need to store them in `static`. Therefore, I deleted them now.

**Pull request merged**